Makefile

```
@@ -1,3 +1,9 @@
+.DEFAULT_GOAL := _setup
+
+
+.PHONY: _setup
+_setup:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)